Added "F Prime" as an acceptable name in the docs

AdvaitDhingra opened the pull request:
| | |
|:---|:---|
|**_Originating Project/Creator_**||
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**||
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**| y|

---
## Change Description

I added "F Prime" in brackets behind F´ in the docs on two occasions. I didn't do it behind every F´, as it would be repetitive. Doing this twice lets the reader know that F Prime is also an acceptable name.

## Rationale

This PR should "fix" what is mentioned in [this](https://github.com/nasa/fprime/issues/526) Issue. This is my first PR, so it seemed like a good place to start.

## Testing/Review Recommendations

This has no effect on the code.

## Future Work

None that I can think of. Let me know if this indication needs to be added elsewhere in the docs.

LeStarch reviewed (approved):
This fixes the issue.  Not sure why the build is cancelling.  I am working on it.

AdvaitDhingra commented:
> This fixes the issue. Not sure why the build is cancelling. I am working on it.

Thanks. I think the resaon may be, that when I made the PR I added an x in some fields, where I shouldn't have. I later edited the description and removed the x's. Maybe that's the reason?

LeStarch commented:
@AdvaitDhingra the header fields are for me such that I can quickly understand the basics before I look through PRs.  For larger PRs this is essential.

The actual build doesn't care (I don't think), and it seems to be failing across different PRs.  Rerunning it seems to fix the issue if another build isn't already running.

AdvaitDhingra commented:
> @AdvaitDhingra the header fields are for me such that I can quickly understand the basics before I look through PRs.  For larger PRs this is essential.
> 
> The actual build doesn't care (I don't think), and it seems to be failing across different PRs.  Rerunning it seems to fix the issue if another build isn't already running.
> 

Ah, I see. I have worked on yaml workflows before, so I can take a look

jsoref commented:
GitHub's Actions were failing site-wide yesterday: https://www.githubstatus.com/incidents/m16jzl31gnqt
